dmenufm

```diff
@@ -533,6 +592,33 @@ dmenufm_action () {
 				rename=
 			fi
 			;;
+		"$FM_ZIP")
+			allowmulti="Bulk Compress"
+			allselection="Bulk Compress all"
+			compression_type=$(printf '%s' "$COMPRESSIONLIST" \
+				| tr ':' '\n' \
+				| verticalprompt "Compression type: " "$FM_ACTION_COLOR_LV1" \
+				|| return \
+				&& allowmulti="NotAllowed")
+			if echo "$compression_type" | grep -E '^tar|^zip|^7z'; then
```

Comment:
May I ask why you need this if condition here?

Reply:
This is here as you are not making file archives with the other commands, and thus you should only be using single files when compressing with bz2 instead of tar.bz2, as one is an archive where you supply multiple files and the other is not.

Reply:
So only formats start as tar, zip and 7z allows multiple files compressed into one file?

Reply:
```sh
$ touch {a..z}
$ ls
a  c  e  g  i  k  m  o  q  s  u  w  y
b  d  f  h  j  l  n  p  r  t  v  x  z
$ bzip2 -k a b c d e f g
$ ls
a      b.bz2  d      e.bz2  g      i  l  o  r  u  x
a.bz2  c      d.bz2  f      g.bz2  j  m  p  s  v  y
b      c.bz2  e      f.bz2  h      k  n  q  t  w  z
$ 
```
As you can see, this will compress each file individually as it is meant for compressing one file, not a directory into an archive or the like.
Maybe we can allow it, but I feel like it may confuse users who don't know better into thinking they are making an archive when it really is gonna make a ton of compressed files like this.